mirror-service/src/main/java/org/hyades/client/OsvClientConfig.java

```diff
@@ -23,8 +23,9 @@ public CloseableHttpClient httpClient() {
     @Named("osvObjectMapper")
     public ObjectMapper objectMapper() {
         ObjectMapper objectMapper = new ObjectMapper()
-                .enable(DeserializationFeature.READ_UNKNOWN_ENUM_VALUES_USING_DEFAULT_VALUE);
+                .enable(DeserializationFeature.READ_UNKNOWN_ENUM_VALUES_USING_DEFAULT_VALUE)
+                .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
```

Comment:
should this not be true?

Reply:
I think ignoring unknown keeps it resilient to failures. If the file we are parsing has fields we are unaware of, we will parse the rest of the fields instead of falling over which I thought was better.